### ISSUE TITLE
handle commands on the client side

### ISSUE THREAD

**Issue by aspeddro** [weilbith/nvim-code-action-menu#49]
Some commands on the `ltex` lsp are resolved on the client side.

Example:

- `_ltex.addToDictionary`
- `_ltex.disableRules`
- `_ltex.hideFalsePositives`

[Docs: ltex commands](https://valentjn.github.io/ltex/ltex-ls/server-usage.html#commands)

```lua
local lspconfig = require 'lspconfig'
local opts = {
  root_dir = function(filename)
    return lspconfig.util.path.dirname(filename)
  end,
  on_init = function(client, _)
    vim.lsp.commands['_ltex.addToDictionary'] = function(cmd, ctx)
      print(vim.inspect { cmd, ctx })
    end
  end,
  settings = {
    ltex = {
      completionEnabled = true,
      language = 'en-US',
      checkFrequency = 'edit',
      setenceCacheSize = 5000,
      additionalRules = {
        enablePickyRules = true,
        motherTongue = 'en-US',
      },
      dictionary = {
        ['en-US'] = { 'Hello', 'Neovim' },
      },
    },
  },
}
lspconfig.ltex.setup(opts)
```

Using `:CodeActionMenu` not run the command `_ltex.addToDictionary`. But using `:lua vim.lsp.buf.code_action()` works.

https://user-images.githubusercontent.com/16160544/160704020-a61c2bf6-e8be-4524-b2fb-eafd4d90c71e.mp4

**Comment by weilbith** [comment]
Yeah, I guess I somehow need to apply [this part](https://github.com/neovim/neovim/blob/75ff156d9b11f23f6156cd6b90fb7bad0a83fd18/runtime/lua/vim/lsp/buf.lua#L513) of what NeoVim core does. Unfortunately it is inside a not accessible function. 😑 
I would be happy if you would open a PR which adds this functionality I linked. You have already the perfect test environment for this "feature", so a contribution would be the optimal case for me. Else I'll try to find some time. But usually if I find the time and motivation to rather work on version 2.0 of this plugin...
